hubble/ui.go

```diff
@@ -199,27 +207,29 @@ func (k *K8sHubble) enableUI(ctx context.Context) (string, error) {
 // 	return nil
 // }
 
-func (k *K8sHubble) generateUICertificate(name string) (corev1.Secret, error) {
-	var (
-		relaySecretFilename string
-	)
-
-	ciliumVer := k.semVerCiliumVersion
-
-	switch {
-	case versioncheck.MustCompile(">1.10.99")(ciliumVer):
-		switch name {
-		case defaults.HubbleUIClientSecretName:
-			relaySecretFilename = "templates/hubble/tls-helm/ui-client-certs.yaml"
-		}
-	}
-
-	relayFile := k.manifests[relaySecretFilename]
-
-	var secret corev1.Secret
-	utils.MustUnmarshalYAML([]byte(relayFile), &secret)
-	return secret, nil
-}
+// TODO we won't generate hubble-ui certificates because we don't want
+//  to give a bad UX for hubble-cli (which connects to hubble-relay)
+// func (k *K8sHubble) generateUICertificate(name string) (corev1.Secret, error) {
+// 	var (
+// 		relaySecretFilename string
+// 	)
+//
+// 	ciliumVer := k.semVerCiliumVersion
+//
+// 	switch {
+// 	case versioncheck.MustCompile(">1.10.99")(ciliumVer):
+// 		switch name {
+// 		case defaults.HubbleUIClientSecretName:
+// 			relaySecretFilename = "templates/hubble/tls-helm/ui-client-certs.yaml"
+// 		}
+// 	}
+//
+// 	relayFile := k.manifests[relaySecretFilename]
+//
+// 	var secret corev1.Secret
+// 	utils.MustUnmarshalYAML([]byte(relayFile), &secret)
+// 	return secret, nil
+// }
```

Comment:
maybe just delete this block of code?

Reply:
I thought about it but there's a lot of code that mentions the "TODO we won't generate hubble-ui certificates because..." it's just a matter of us deciding to have it back and uncomment it. I figure that it would better to leave the commented code than having someone spending a couple hours to figure out how to add it back.